x-pack/test/functional/services/ml/data_frame_analytics_creation.ts

```diff
@@ -236,6 +237,76 @@ export function MachineLearningDataFrameAnalyticsCreationProvider(
       );
     },
 
+    async assertRuntimeMappingSwitchExists() {
+      await testSubjects.existOrFail('mlDataFrameAnalyticsRuntimeMappingsEditorSwitch');
+    },
+
+    async assertRuntimeMappingEditorExists() {
+      await testSubjects.existOrFail('mlDataFrameAnalyticsAdvancedRuntimeMappingsEditor');
+    },
+
+    // async enableRuntimeMappingsEditor() {
```

Comment:
Should we get rid of this commented code?

Reply:
Oh good catch - thanks 68f885778a2398339acea029f5ab556e13ed9124